Author: Diane Trout <diane@ghic.org>
Subject: remove --cov-config option from pytest it doesn't seem to work
 on Debian, and coverage isn't that important for our tests anyway.
Forwarded: no

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -105,7 +105,6 @@
 --ignore=.github
 --strict-markers
 --strict-config
---cov-config=pyproject.toml
 -p no:asyncio
 -p no:legacypath'''
 filterwarnings = [
